api/src/backend/api/specs/v1.yaml

```diff
@@ -3543,7 +3564,7 @@ paths:
     get:
       operationId: overviews_providers_retrieve
       description: Retrieve an aggregated overview of findings and resources grouped
-        by providers. The response includes the count of passed, failed, and manual
+        by providers. The response includes the count of passed, failed, and muted
```

Comment:
Probably manual was right as it is valid status. However it's deprecated and no more checks will generate findings with that status.